### PR TITLE
cls: add Thai translation

### PR DESCRIPTION
**Pull request by MasterIceZ**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.th/windows/cls.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# cls
+
+> ล้างตัวอักษรทั้งหมดที่อยู่บนหน้าต่าง
+> ข้อมูเพิ่มเติ่ม: https://docs.microsoft.com/windows-server/administration/windows-commands/cls
```
Comment (kbdharun) [inline pages.th/windows/cls.md:4]:
```suggestion
> ข้อมูเพิ่มเติ่ม: https://learn.microsoft.com/windows-server/administration/windows-commands/cls
```
Reference: #8543, There is an ongoing PR to update all Microsoft links to `learn.microsoft.com`. The original page will be updated soon.

**Review by kbdharun — APPROVED** [review]
Welcome to tldr @MasterIceZ. Thanks for your contribution, I have a suggestion that needs to be addressed. 

Also a quick doubt, `ข้อมูเพิ่มเติ่ม` means `additional information` right? 
And ` ดูเพิ่มเติม` means `See also` right? 
If you could clarify this, all Thai pages can have a uniform term because currently, all pages use `ดูเพิ่มเติม` but yours seems to be accurate.

> Note: I am not a native speaker and I use Google Translate to validate the translations.

**Comment by MasterIceZ** [comment]
Yes. ``ข้อมูเพิ่มเติม`` means ``additional information``.

**Comment by kbdharun** [comment]
> Yes. `ข้อมูเพิ่มเติม` means `additional information`.

Thanks for the information. I will update the other pages in `pages.th` with this one then.

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**